elements/pfe-icon/README.md

```diff
@@ -88,8 +88,21 @@ To updating an existing icon set, you use the same `addIconSet` function.  The f
 
 There are several powerful variables available to hook into and override default styles.
 
-- Color: the `color` attribute is available to pull icon color from your theming layer.  For more fine-grained control, `--pfe-icon--color` is available to override the color of a specific icon or sets of icons and will be applied to either the SVG lines or the background of the circle (if circled).  [Examples][color-examples]
-- Background color: the `color` attribute is available to pull background color from your theming layer.  For more fine-grained control, `--pfe-icon--BackgroundColor` is available to override the background color of a specific icon or sets of icons.  Be sure to set `--pfe-icon--context` to the appropriate context if you are setting the background-color.
+- Color: the `color` attribute is available to pull icon color from your theming layer.  For more fine-grained control, `--pfe-icon--color` is available to override the color of a specific icon or sets of icons and will be applied to the SVG.  [Examples][color-examples]
+- Background color: the `color` attribute is available to pull background color from your theming layer.  For more fine-grained control, `--pfe-icon--BackgroundColor` is available to override the background color of a specific icon or sets of icons.  Be sure to set `--pfe-icon--context` to the appropriate context if you are setting the background-color or use the more fine-grained `--pfe-icon--color` to set a specific color on the SVG lines.
+
+| Theme hook | Description | Default |
```

Comment:
Do we consistently say "Theme hook" in the docs? I wonder if these should be more explicit & say "CSS Custom Properties" or "CSS Variables"

Reply:
I opened a story to address this because these tables are standard in all our documentation and if we update one we should probably update all at the same time 👍 https://github.com/patternfly/patternfly-elements/issues/1632